crates/pallets/starknet/src/lib.rs

```diff
@@ -185,9 +199,12 @@ pub mod pallet {
                 PALLET_STARKNET_SCHEMA,
                 &StarknetStorageSchema::V1,
             );
+
             for (address, class_hash) in self.contracts.iter() {
                 ContractClassHashes::<T>::insert(address, class_hash);
             }
+
+            LastKnownEthBlock::<T>::set(Some(16868366));
```

Comment:
Because we still don't have the config and it was the latest block when i coded it

Reply:
Ok but I am not comfortable keeping this. It can lead to random behaviour. Maybe we should push the code but completely disable the offchain worker for now.

Reply:
Yes we should do that otherwise it'll try to execute all the mainnet messages at each block which we don't want